modules/emulator/R/minimize.GP.R

```diff
@@ -227,7 +230,7 @@ mcmc.GP <- function(gp, x0, nmcmc, rng, format = "lin", mix = "joint", splinefcn
       currY <- get.y(gp, xcurr, n.of.obs, llik.fn, priors, settings)
       ycurr <- currY$posterior.prob
       pcurr <- currY$par
-      newY  <- get.y(gp, xnew, n.of.obs, llik.fn, priors, settings)
+      newY  <- get.y(gp, xnew, n.of.obs, llik.fn, priors, settings, currY$llik.par)
```

Comment:
Here and below, how does llik.par end up in currY$llik.par if it's not passed in a few lines above in the ```currY <- get.y``` line?

Wait, I just realized that you steathly snuck in the Gibbs update **before** the M-H update by burying it in the function called by get.y -- this is going to be really unintuitive to folks, so I'd add some inline documentation to the line above making this explicit.